### PR TITLE
added python 3.8 in test matrix

### PR DESCRIPTION
**Pull request by iamsobanjaved**
- Added python3.8 travis worker
- Added Django 3.0 in tox file for testing
- Removed support for python < 3.5 and Django < 2.2 

https://openedx.atlassian.net/browse/BOM-1574

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 from setuptools import setup
 
 
-version = '1.0.1'
+version = '1.1.1'
```
Comment (morenol) [inline setup.py:8]: I think that this should be 1.1.0

File: .travis.yml [hunk 2]
```diff
@@ -1,34 +1,40 @@
...
-  - TOXENV=eslint
-  - TOXENV=pylint
-  - TOXENV=csslint
-  - TOXENV=pycodestyle
-  - TOXENV=django111
-  - TOXENV=django20
-  - TOXENV=django21
   - TOXENV=django22
 
+matrix:
```
Comment (mraarif) [inline .travis.yml:13]: why not run all these checks against both python versions?

File: setup.py (hunk 1 above)
Comment (iamsobanjaved) [reply setup.py]: Updated

File: .travis.yml (hunk 2 above)
Comment (iamsobanjaved) [reply .travis.yml]: Updated

**Pull request merged**